src/client/components/menu/Menu.styles.css

```diff
@@ -0,0 +1,96 @@
+header {
+  justify-content: space-between;
+  background-color: black;
+  height: fit-content;
+  overflow: hidden;
+}
+
+header #login {
+  padding: 1em;
+}
+
+header .user-icon {
+  float: left;
+  cursor: pointer;
+}
+
+header .icons {
+  float: right;
+  margin-right: 1em;
+  justify-content: space-around;
+  cursor: pointer;
+}
+
+header .logo_container {
+  background-color: white;
+  display: block;
+  padding-bottom: 3em;
+}
+
+header img {
+  height: 2em;
+  width: 2em;
+}
+
+header .navbar {
+  overflow: hidden;
+  background-color: #333;
+}
+
+/* lis inside the navbar */
+header .navbar li {
+  float: left;
+  font-size: 1.2em;
+  color: white;
+  text-align: center;
+  padding: 0.8em 0.8em;
+  cursor: pointer;
+}
+
+/* The dropdown container */
+header .dropdown {
+  margin-left: 20%;
+  float: left;
+  overflow: hidden;
+}
+
+/* Dropdown button */
+header .dropbtn {
+  font-size: 1.2em;
+  border: none;
+  color: white;
+  padding: 0.8em 0.8em;
+  background-color: inherit;
+  cursor: pointer;
+}
```

Comment:
This is a small thing, but I see a fair amount of duplication in your CSS. For example you are declaring `font-size: 1.2em;` on both `header .dropbtn` and `header .navbar li`. Perhaps you could just move it to `header` so it trickles down to the other elements. Same with padding and if you go through your other changes you can probably find more examples. It's not critical for mergin this PR, though.